### ISSUE TITLE
Multiply overlapping rectangles in one of EA output files

### ISSUE THREAD

**Issue by maxirmx** [metanorma/libemf2svg#8]
[issue-7.zip](https://github.com/metanorma/libemf2svg/files/6743832/issue-7.zip)

**Comment by maxirmx** [comment]
emf file actually conatains three rectangles with different styles in for each entity 
However, png file generated by EA shows only one of them 
There may be an issue with Z-order or filling or clipping in svg file

**Comment by ronaldtse** [comment]
@maxirmx Yes please help look at this issue. I also noticed that the rectangle corner radius is also different for this image.

**Comment by maxirmx** [comment]
There are two issues:
- diameters for ellipses of rounded rectangle corners are processed incorrectly (fixed at https://github.com/maxirmx/libemf2svg/commit/db740ad4fa659f61b646b55d9de5c20ceea63d03)
- rounded rectangle is overlapped by straight rectangle under yet unknown conditions

**Comment by ronaldtse** [comment]
Thanks @maxirmx , once fixed let's PR upstream.

**Comment by maxirmx** [comment]
[Primitives of Temporal Geometry and Topology.zip](https://github.com/metanorma/libemf2svg/files/6760496/Primitives.of.Temporal.Geometry.and.Topology.zip)

Attached archieve is the single  case when rounded rectangle is overlapped by straight rectangle.  However, svg file is correct translation of emf which really contains overlapping rectangles as opposed to png file.

It is possible that SA applies some custom logic when  exporting to png. 
In any case more samples are required to continue work on this problem.
